src/meta/src/hummock/hummock_manager.rs

```diff
@@ -262,6 +265,9 @@ where
             compaction_scheduler: parking_lot::RwLock::new(None),
             compactor_manager,
             max_committed_epoch: AtomicU64::new(0),
+            // TODO: assign worker id for each meta node in meta node cluster, so that generated id
+            // is unique within the cluster.
+            sst_id_generator: Mutex::new(IdGenerator::with_epoch(0, *UNIX_SINGULARITY_DATE_EPOCH)),
```

Comment:
Can we just generate id in client side with different worker id, so we don't need a extra RPC call.

Reply:
Then each client should be assigned a worker id for its generator, which is different from the cluster node id?